migrator/src/main/java/com/google/phonenumbers/migrator/MigrationFactory.java

```diff
@@ -56,21 +56,23 @@ public static MigrationJob createMigration(String number, String country) throws
         .orElseThrow(() -> new RuntimeException(
             "Country code " + countryCode+ " not supported in metadata"));
 
-    return new MigrationJob(numberRanges, countryCode, recipes, ranges);
+    return new MigrationJob(numberRanges, countryCode, recipes, ranges, false);
```

Comment:
When having a parameter that isn't clear, such as false, it is always good to add an inline comment, ie:
return new MigrationJob(numberRanges, countryCode, recipes, ranges, /* parameterName= */ false);